modules/javafx.graphics/src/main/java/javafx/css/converter/EffectConverter.java

```diff
@@ -70,8 +78,16 @@ public String toString() {
         return "EffectConverter";
     }
 
+    /**
+     * Converter to convert DropShadow {@code Effect}
+     * @since 9
```

Comment:
This can only be checked manually.
I have checked and added the @since tag to these *Converter nested classes.

Reply:
I ran a script to check for missing `@since` tags. The following class are still missing them:

```
DeriveColorConverter
DeriveSizeConverter
StopConverter
StringConverter.SequenceConverter
```

Reply:
Ah.. Thanks for running the script. It is tough to find it manually. I will fix these soon.